scripts/scil_compute_msmt_frf.py

```diff
@@ -0,0 +1,250 @@
+             'inside the GM mask defined by mask_gm. Each voxel below this '
+             'threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--fa_thr_csf', default=0.1, type=float,
+        help='If supplied, use this threshold to select CSF voxels from the '
+             'FA inside the CSF mask defined by mask_csf. Each voxel below '
+             'this threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--md_thr_gm', default=0.0007, type=float,
+        help='If supplied, use this threshold to select GM voxels from the MD '
+             'inside the GM mask defined by mask_gm. Each voxel below this '
+             'threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--md_thr_csf', default=0.003, type=float,
+        help='If supplied, use this threshold to select CSF voxels from the '
+             'MD inside the CSF mask defined by mask_csf. Each voxel below '
+             'this threshold will be selected. [%(default)s]')
+
+    p.add_argument(
+        '--min_nvox', default=100, type=int,
+        help='Minimal number of voxels needed for each tissue masks '
+             'in order to proceed to frf estimation. [%(default)s]')
+    p.add_argument(
+        '--tolerance', type=int, default=20,
+        help='The tolerated gap between the b-values to '
+             'extract\nand the current b-value. [%(default)s]')
+    p.add_argument(
+        '--dti_bval_limit', type=int, default=1200,
+        help='The highest b-value taken for DTI. [%(default)s]')
+    p.add_argument(
+        '--roi_radii', default=[10], nargs='+', type=int,
+        help='If supplied, use those radii to select a cuboid roi '
+             'to estimate the response functions. The roi will be '
+             'a cuboid spanning from the middle of the volume in '
+             'each direction with the different radii. The type is '
+             'either an int or an array-like (3,). [%(default)s]')
+    p.add_argument(
+        '--roi_center', metavar='tuple(3)', nargs=3, type=int,
+        help='If supplied, use this center to span the cuboid roi '
+             'using roi_radii. [center of the 3D volume]')
+
+    p.add_argument(
+        '--wm_frf_mask', metavar='file', default='',
+        help='Path to the output WM frf mask file, the voxels used '
+             'to compute the WM frf.')
+    p.add_argument(
+        '--gm_frf_mask', metavar='file', default='',
+        help='Path to the output GM frf mask file, the voxels used '
+             'to compute the GM frf.')
+    p.add_argument(
+        '--csf_frf_mask', metavar='file', default='',
+        help='Path to the output CSF frf mask file, the voxels used '
+             'to compute the CSF frf.')
+
+    p.add_argument(
+        '--frf_table', metavar='file', default='',
+        help='Path to the output frf table file. Saves the frf for '
+             'each b-value, in .txt format.')
```

Comment:
@karanphil can you do a bit of comestics on the -h. i.e. put in some newlines whereever the text is too long please. It is not "pretty" or pleasant to read. Maybe you could try not to be wider than
"Path to the input WM mask file, used to improve the final WM frf mask."

![Screen Shot 2021-01-12 at 8 11 02 PM](https://user-images.githubusercontent.com/2170887/104393395-60fe8f80-5512-11eb-8413-f9af336445f5.png)